style/properties/longhands/position.mako.rs

```diff
@@ -183,34 +168,6 @@ ${helpers.single_keyword(
         spec="https://github.com/w3c/csswg-drafts/issues/4650",
         affects="layout",
     )}
-
-    ${helpers.predefined_type(
-        "align-items",
-        "AlignItems",
-        "specified::AlignItems::normal()",
-        engines="gecko",
-        spec="https://drafts.csswg.org/css-align/#propdef-align-items",
-        extra_prefixes="webkit",
-        animation_value_type="discrete",
-        servo_restyle_damage="reflow",
-        affects="layout",
-    )}
-
-    #[cfg(feature = "gecko")]
-    impl_align_conversions!(crate::values::specified::align::AlignItems);
-
-    ${helpers.predefined_type(
-        "justify-items",
-        "JustifyItems",
-        "computed::JustifyItems::legacy()",
-        engines="gecko",
-        spec="https://drafts.csswg.org/css-align/#propdef-justify-items",
-        animation_value_type="discrete",
-        affects="layout",
-    )}
-
-    #[cfg(feature = "gecko")]
-    impl_align_conversions!(crate::values::specified::align::JustifyItems);
```

Comment:
I think it's better not to adjust the position of these, as this makes the diff with upstream harder to process.

Reply:
I think I've put them back in the order they were in.